src/artemis/experiment_plans/full_grid_scan.py

```diff
@@ -74,10 +75,10 @@ def get_plan(
         "snap_2_filename": os.path.basename(os.path.abspath(gda_snap_2)),
     }
 
-    oav_params = OAVParameters("xrayCentring")
+    oav_params = OAVParameters("loopCentring", **oav_param_files)
```

Comment:
Should: I think there is an issue using `loopCentring` here, spun out into https://github.com/DiamondLightSource/python-artemis/issues/653